src/lambda/explorer/defaultRegionNode.ts

```diff
@@ -5,7 +5,7 @@
 
 import { TreeItemCollapsibleState } from 'vscode'
 import { RegionInfo } from '../../shared/regions/regionInfo'
-import { AWSTreeNodeBase } from '../../shared/treeview/awsTreeNodeBase'
+import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
```

Comment:
This file should prolly be moved out of the lambda/ folder too

Reply:
Can do.

Reply:
✔️